deploy/setup_test.sh

```
@@ -0,0 +1,171 @@
+#
+# Run:
+#   ./deploy/setup_test.sh
+#
+# Exits non-zero on any failure. Prints PASS / FAIL per case.
+#
+# rc11 #80: covers the disable-clears path that the review caught
+# as silently no-op'ing in the previous cut.
+
+set -euo pipefail
+
+SCRIPT_DIR_REAL="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] SCRIPT_DIR_REAL appears unused. Verify use (or export if used externally).

(SC2034)